### PR TITLE
Objects are saved in a directory instead of root

### PR DESCRIPTION
**Pull request by marijnvanh**
## Description:

Objects are saved in a directory instead of root

**Related issue (if applicable):** fixes #<issue number goes here>

## Checklist:
  - [ ] The code change works
  - [ ] Passes all tests: `make test`
  - [ ] There is no commented out code in this PR.
  - [ ] `norminette srcs libft | grep -E "^Error" | wc -l` is not higher than master. If it is, run `norminette srcs libft | grep -E "^Error" -B 1` to see errors
  - [ ] I solemny swear my code is compliant with the [README][readme-file]

[readme-file]: https://github.com/OscarMulder/codam-42sh/blob/master/README.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -123,14 +126,14 @@ test_norm: fclean
 	@sh ${GITHUB_WORKSPACE}/test/norminette.sh
 
 $(TESTOBJECTS): $(TESTS)
-	@$(CC) $(FLAGS) $^ $(INCLUDES) $(CRITERIONINCLUDES) -c
+	@$(CC) $(FLAGS) $^ $(INCLUDES) $(CRITERIONINCLUDES) -c
```
Comment (thijsdejong) [inline Makefile]: Space at end of the line

File: Makefile [hunk 2]
```diff
@@ -80,33 +81,35 @@ builtin_fc_print_helpers builint_fc_find_index builtin_fc_substitute \
...
 	@$(MAKE) -C libft
 
+$(OBJDIR):
+	@mkdir $(OBJDIR)
+
 clean:
-	@rm -f $(OBJECTS) $(TESTOBJECTS) main.o
-	@$(MAKE) -C libft clean
+	@rm -f $(OBJECTS) $(TESTOBJECTS) $(OBJDIR)main.o
+	@$(MAKE) -C libft fclean
```
Comment (thijsdejong) [inline Makefile]: Why this change?

Comment (jbrinksma) [reply Makefile]: To paraphrase @marijnvanh: "We want to clean all dependencies of vsh and libft is one of those"
